docs/reference/action_create_edit.rst

```diff
@@ -69,14 +69,12 @@ For more detailed information about routes, see :doc:`routing`.
 
         protected function configureRoutes(RouteCollection $collection): void
         {
-            /* Removing the edit route will disable editing entities. It will also
-             * use the 'show' view as default link on the identifier columns in the list view.
-             */
+            // Removing the edit route will disable editing entities. It will also
+            // use the 'show' view as default link on the identifier columns in the list view.
```

Comment:
What's wrong with using multiline comments? It's a multiline comment, not 2 single lines comments… but it's not phpdoc either, so I would do this:
```
/* Removing the edit route will disable editing entities. It will also
use the 'show' view as default link on the identifier columns in the list view. */
```

Reply:
done